src/lib/views/QueueView.svelte

```diff
@@ -522,9 +522,10 @@
     }
 
     function onRightClick(e, song, idx) {
-        if (!songsHighlighted.find((s) => s.viewModel.index === idx)) {
-            highlightSong(song, idx, false);
-        }
+        songsHighlighted = [song];
```

Comment:
This breaks the right-click behaviour with multi-select, so you would no longer be able to remove multiple tracks from the queue.

Reply:
Oops! Sorry. It's fixed.